### ISSUE TITLE
How to correctly link tuples

### ISSUE THREAD

**Issue by jakearchibald** [whatwg/infra#234]
https://infra.spec.whatwg.org/#tuple

In the example:

> A status is an example tuple consisting of a code (a three-digit number) and text (a byte sequence).

I would expect "status" to be a top-level definition, and "code" and "text" to be child definitions of "status". (I'd also expect byte sequence to link to its definition).

> Let statusInstance be the status (200, `OK`).

I'd expect "status" to link back to its definition.

> If statusInstance’s code is 404, then

I'd expect "code" to link back to its definition.

I'm currently reviewing a PR that uses tuples in this no-linking way & it feels difficult to read and navigate, but infra appear to (accidentally) recommend this style.

Is it deliberate that these aren't linked?

**Comment by annevk** [comment]
No, I think I didn't feel the need at the time to do it for the example and reviewers didn't notice it or care strongly either. I regret the error.

**Comment by jakearchibald** [comment]
One of the things I've never been sure of when writing examples like this: is there a way to say "this is just an example, not a real definition" in a way that Bikeshed understands?

**Comment by annevk** [comment]
That's a good question and might well be why I didn't do it. I thought we used `<dfn>` elsewhere in examples and we haven't.

Perhaps @tabatkins has ideas. I guess we could use `for=example` and `noexport` or some such, but it'd be a little ugly.

**Comment by jakearchibald** [comment]
I guess the tricky part is if you have two examples that contain the same example definition.

**Comment by tabatkins** [comment]
A `<dfn lt>` (empty `lt` attribute) marks it as not a real definition. But then you can't link to it at all. I prefer just using `<b id>` and a manual `<a>` if you're doing pure examples.

**Comment by privsenotes** [comment]
Stop thies hacker these steling form gov
